src/mn_setup.c

```diff
@@ -2836,12 +2825,24 @@ static setup_menu_t gen_settings3[] = {
     {"Acceleration", S_THERMO, CNTR_X, M_THRM_SPC, {"mouse_acceleration"},
      .strings_id = str_mouse_accel, .action = G_UpdateMouseVariables},
 
+    MI_GAP,
+
+    // [FG] double click to "use"
+    {"Double-Click to \"Use\"", S_ONOFF, CNTR_X, M_SPC, {"dclick_use"}},
+
+    {"Free Look", S_ONOFF, CNTR_X, M_SPC, {"mouselook"},
```

Comment:
Just out of curiosity, why did you decide to move ‘Free Look’ to the bottom? I suspect that many users will be looking for it in the menu.

Reply:
I think turn sensitivity is the most important setting for both mouse and gamepad, so it should be the first thing a player sees when moving through the tabs. Free look is less important by comparison. In my opinion, y-shearing is not great to use and players looking for true free look aren't using Woof anyway.

Reply:
That said, I'm okay with moving it back if it seems too disruptive.

Reply:
> That said, I'm okay with moving it back if it seems too disruptive.

No, looks good as it is.